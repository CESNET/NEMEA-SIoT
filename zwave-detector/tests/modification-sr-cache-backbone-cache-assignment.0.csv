time,time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2020-01-19T22:57:00,2020-01-19T22:57:00.000,1,00
2020-01-19T22:57:02,2020-01-19T22:57:02.000,1,e145347301410611020114010205031049
2020-01-19T22:57:02,2020-01-19T22:57:02.000,1,e14534730203060a0110
