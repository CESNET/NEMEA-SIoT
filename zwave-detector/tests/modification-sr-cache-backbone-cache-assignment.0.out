2,2020-01-19T22:57:02.000,3,"Unknown nodes in SR Cache: 5,."
